src/VectorTypes.f90

```diff
@@ -20,6 +23,7 @@
 !>  - @ref ExceptionHandler "ExceptionHandler": @copybrief ExceptionHandler
 !>  - @ref Allocs "Allocs": @copybrief Allocs
 !>
+!> TIBWSFB: This example doesnt match the current interface
```

Comment:
I'm writing new examples, but I kind of got stuck holding the bag on this; these should have been updated when the original %init routines were changed to take parameter lists instead of a list of arguments. Also, for performance reasons), I don't think we should be using parameter lists anyways  (e.g. where frequently constructing matrices is needed, as in the previous iteration of the Sundials interface. Parameter lists are dreadfully slow, and doing plist validation is especially so. I still advocate for a simple derived type for storing arguments, where each optional argument is initialized with a sane default. Discussion for another day.